### PR TITLE
Tap with 2 fingers to show the search dialog

### PR DESCRIPTION
**Pull request by agrogers**
I have always thought that the search button is too far away from the my normal workflow. Going into the hamburger menu just felt wrong and slow (that does go to show how lazy I have become!).  A two finger tap is unused (as far as I am aware) and is a nice shortcut for this. I think we could also have 3 finger taps but that will require a fancier implementation that I have done. Maybe there is justification to allow the user to customise what a two finger tap does. 

This partially addresses #2402

### CODE REVIEW

**Comment by tuomas2** [comment]
Common users will never ever figure this out... but I'll give it a try.

File: app/src/main/java/net/bible/android/view/activity/page/BibleView.kt [hunk 1]
```diff
@@ -686,7 +685,7 @@ class BibleView(val mainBibleActivity: MainBibleActivity,
             "?lang=$lang&fontModuleNames=$fontModuleNames&styleModuleNames=$styleModuleNames&featureModuleNames=$featureModuleNames&rtl=$isRtl&night=$nightMode")
     }
 
-     fun onEvent(e: ReloadAddonsEvent) {
+     fun onEvent() {
```
Comment (tuomas2) [inline app/src/main/java/net/bible/android/view/activity/page/BibleView.kt]: I think this change is unintentional? Breaks reloading addons...

Comment (tuomas2) [reply app/src/main/java/net/bible/android/view/activity/page/BibleView.kt]:
```suggestion
     fun onEvent(e: ReloadAddonsEvent) {
```

**Comment by tuomas2** [comment]
To avoid merge conflict and make reviewing easier, please do not use auto-reformat in feature / bug fix branches in general. It's ok to do reformatting separate PR's though.

File: app/src/main/java/net/bible/android/view/activity/page/BibleView.kt (hunk 1 above)
Comment (agrogers) [reply app/src/main/java/net/bible/android/view/activity/page/BibleView.kt]: > I think this change is unintentional? Breaks reloading addons...
Yeah, good find. Thanks.

**Comment by agrogers** [comment]
> Common users will never ever figure this out... but I'll give it a try.

It's as findable as the swipe left, right and down on the Verse Selector I think. They all need introducing. I would love to include one of those guided tutorials for new users and when new features are added. Or even just a text message...

**Comment by tuomas2** [comment]
I'm having many accidental two finger taps with my tablet that I use with two hands...

**Comment by agrogers** [comment]
I haven't had any issues with it yet. But I only use my phone. I liked it
when I have needed. I wouldn't see any need for it on a tablet because you
have the search button (I think) in the action bar. I think that is the
best solution to my problem, but there is no room left for a search button
on my phone.

**Comment by timbze** [comment]
I can see the handiness of this... but it also feels like an unnatural shortcut... Not that I have a problem with it necessarily though. This feels like a feature that should be enabled by an advanced setting that regular users never see.

**Comment by tuomas2** [comment]
I agree with Timmy, this is not good to be enabled by default.

**Comment by tuomas2** [comment]
Created issue for this #2464.

**Comment by tuomas2** [comment]
I already had forgotten about this and have been recently wondering why I'm always going to search unintentionally.  Weird behavior. I think I must revert this (at least until #2464 is done).

**Pull request merged**